shared/reducers/fs.tsx

```diff
@@ -57,123 +56,28 @@ const initialState: Types.State = {
 export const _initialStateForTest = initialState
 
 const updatePathItem = (
```

Comment:
wow so much deleted code 👌

Reply:
Yeah I talked to Nojima and he thought it wouldn't be worth it to do those isEqual checks trying to avoid setting the same value. I think it doesn't matter as much as before now that we have subscriptions based loading.